packages/react/index.js

```diff
@@ -13,5 +13,27 @@ module.exports = {
             'error',
             4,
         ],
+        'react/forbid-prop-types': [
+            'error',
+            {
+                'forbid': ['any']
+            }
+        ],
+        'react/prop-types': [
+            'error',
+            {
+                ignore: ['styles']
+            }
+        ],
+        'import/no-extraneous-dependencies': [
+            'error',
+            {
+                'devDependencies': [
+                    '**/*.test.js',
+                    '**/*.test.jsx',
+                    'src/setupTests.js',
```

Comment:
maybe this file is project specific?

Reply:
Regarding the object-curly-newline rule: What's the specific advantage of allowing one more property compared to deviating from the airbnb standard?

Regarding the setupTests file: I thought about it, but it's a pretty common file when using create-react-app, isn't it? So I thought it might make sense to exclude it here, as to reduce the number of potential project specific overrides needed.

Reply:
I don't want to say, four is better than three. I only say this changed and a lot of checks will fail…